recipes/kubernetes/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "kubernetes"%}
+{% set version = "1.7.14"%}
+
+{% set sha256 = "1f3c05e60882f014236a1c93fac05afde49b0fcb67aabb4df0a11b96b5aa5d24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kubernetes/{{ name }}/archive/v{{ version }}.tar.gz
+  fn: {{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [not linux]
```

Comment:
osx doesn't work?

Reply:
Not for the hypercube build. I can investigate building the client side-only for OSX. Can I do that after the merge? I usually have better luck using the CI servers that way.

Reply:
sure